Providers/nxOMSAutomationWorker/automationworker/worker/curlhttpclient.py

```diff
@@ -175,6 +176,9 @@ def build_request_cmd(self, url, headers, method=None, data_file_path=None):
         cmd.append(url)
         return cmd
 
+    def os_is_redhat():
+        return os.path.exists("/etc/redhat-release")
```

Comment:
Is path.exists case insensitive?  Or, is path in Linux always lowercase?  IOW, does case matter in this or do you need to convert to lowercase before checking?

Reply:
this path is always lowercase